src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,50 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`.
+
+To meet CIS compliance, admin.conf permissions are changed from the default `root:sudo 440` to `root:root 444`.
```

Comment:
Should this be moved under known limitations?

Reply:
OK.